lib/src/anchor.dart

```diff
@@ -38,8 +39,7 @@ class Anchor {
   /// fractional representation.
   Vector2 get toVector2 => Vector2(x, y);
```

Comment:
since you are on it can we change this to be a function?

Reply:
Done!